lib/compat/wordpress-6.0/class-gutenberg-rest-global-styles-controller.php

```diff
@@ -231,7 +231,11 @@ public function get_theme_items( $request ) {
 			foreach ( $nested_html_files as $path => $file ) {
 				$decoded_file = wp_json_file_decode( $path, array( 'associative' => true ) );
 				if ( is_array( $decoded_file ) ) {
-					$variations[] = ( new WP_Theme_JSON_Gutenberg( $decoded_file ) )->get_raw_data();
+					$variation = ( new WP_Theme_JSON_Gutenberg( $decoded_file ) )->get_raw_data();
+					if ( ! isset( $variation['name'] ) ) {
+						$variation['name'] = basename( $path, '.json' );
```

Comment:
Not sure the name of the file is quite right for using in this case. Maybe mark it as "No name provided"? That would make more sense to me than hearing the name of the file.